scipy/stats/tests/test_discrete_distns.py

```diff
@@ -96,3 +98,12 @@ def test_betabinom_bernoulli():
     p = betabinom(1, a, b).pmf(k)
     expected = bernoulli(a / (a + b)).pmf(k)
     assert_almost_equal(p, expected)
+
+
+def test_zipfian():
+    # test limiting case that zipfian(a, n) -> zipf(a) as n-> oo
+    a = 2.5
+    N = 10000000
+    k = np.arange(1, 21)
+    assert_allclose(zipfian.pmf(k, a, N), zipf.pmf(k, a))
+    assert_allclose(zipfian.cdf(k, a, N), zipf.cdf(k, a))
```

Comment:
I can add tests to test against some other reference.

Reply:
zipfUC from R's VGAM package seems innacurate for low `a`, `N`. Trying this at [rdrr.io](https://rdrr.io/cran/VGAM/man/zipfUC.html):
```
library(VGAM)
print(dzipf(0:21, shape = 1.01, N = 20))
print(sum(dzipf(0:21, shape = 1.01, N = 20)))
print(pzipf(20, shape = 1.01, N = 20))
```
The first line prints the pmf, the second is the sum of the pmf, and the third is the cdf. The sum of the pmf values is not 1.